mutiny-server/src/main.rs

```diff
@@ -0,0 +1,103 @@
+#![allow(incomplete_features)]
+
+mod config;
+mod extractor;
+mod routes;
+mod sled;
+
+use crate::config::Config;
+use crate::sled::SledStorage;
+use axum::http::{StatusCode, Uri};
+use axum::routing::{get, post};
+use axum::{Extension, Router};
+use bitcoin::bip32::ExtendedPrivKey;
+use clap::Parser;
+use log::{debug, info};
+use mutiny_core::storage::MutinyStorage;
+use mutiny_core::{generate_seed, MutinyWalletBuilder, MutinyWalletConfigBuilder};
+use shutdown::Shutdown;
+use std::time::Duration;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    pretty_env_logger::try_init()?;
+    let mut shutdown = Shutdown::new().unwrap();
+    let config: Config = Config::parse();
+
+    let network = config.network();
+    let storage = SledStorage::new(&config.db_file, config.clone().password)?;
```

Comment:
can instead do `config.password.clone()` here, that way you don't need to clone the entire config